src/schemas/json/substreams-manifest-schema.json

```
@@ -0,0 +1,173 @@
+{
+  "name": "Substreams Manifest Schema",
+  "description": "A Schema to support in making Substreams manifest files",
+  "type",: "object",
+  "properties" : {
+    "specVersion" : {
+      "enum" : ["v0.1.0"]
```

```suggestion
      "description": "A specification version",
      "type": "string",
      "const": "v0.1.0"
```

Required fix: add `description` and `type` keys
Optional fix: `enum` can be replaced with `const` when one item is listed